test/e2e/storage/drivers/csi.go

```diff
@@ -310,8 +306,11 @@ func (g *gcePDCSIDriver) GetClaimSize() string {
 	return "5Gi"
 }
 
-func (g *gcePDCSIDriver) CreateDriver() {
+func (g *gcePDCSIDriver) CreateDriver(f *framework.Framework) (*testsuites.PerTestConfig, func()) {
+	framework.SkipIfMultizone(f.ClientSet)
```

Comment:
why did we move `SkipIfMultizone` from `SkipUnsupportedTest` to `CreateDriver`? It seems like it would belong in `SkipUnsupportedTest`

Reply:
Because `SkipUnsupportedTest` now runs before `framework.Framework` is initialized. Cheap tests that just check `e2e.test` parameters (like cloud provider) can and should be done in `SkipUnsupportedTest` to avoid setting up more expensive resources like `framework.Framework`. Tests that depend on a connection to the cluster can only be done once the framework is ready.

Reply:
lets document that entire statement in a comment so other dev's (probably a forgetful me) don't waste time trying to move that line back to `SkipUnsupportedTest` and having to figure out why it isn't working

Reply:
If you had attempted that, you would have found that you can't even get it compiled, because `SkipUnsupportedTest` doesn't have a `framework.Framework` pointer. But yes, let's document this better.

Reply:
Done.